components/prism-diff.js

```diff
@@ -39,11 +39,24 @@
 			alias.push("bold");
 		}
 
-		Prism.languages.diff[name] = {
-			// pattern: /^(?:[_].*(?:\r\n?|\n|(?![\s\S])))+/m
-			pattern: RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm'),
-			alias: alias
-		};
+		var pattern = RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm');
+
+		if (['-', '<', '+', '>'].indexOf(prefix) !== -1) {
+			var inside = {
+				prefix: RegExp('^[' + prefix + ']', 'm')
+			};
```

Comment:
This inside won't work if the next character after the prefix is the same character as the prefix.
Example:

```
++
--
```

```suggestion
			var inside = {
				'line': {
				    pattern: /^(.).+(?:\r\n?|\n)?/,
				    lookbehind: true
				},
				'prefix': /[\s\S]/
			};
```

Reply:
Sorry, I don't quite understand the suggested change here, it wraps every single character with `prefix` now which is not really what I want:

![image](https://user-images.githubusercontent.com/1091472/78439015-ef073900-76a9-11ea-918f-d9cc2eb6e8f4.png)

Reply:
Please also look at the `line` token. The trick is that the line token will capture every line (plus line ends) but leaves out the prefix. So the only thing left is the prefix.

Reply:
Thank u! It's my fault to put `prefix` key before `line`.